package.json

```diff
@@ -91,9 +89,11 @@
     "eslint-plugin-react": "7.12.4",
     "eslint-plugin-react-hooks": "^1.6.0",
     "eslint-plugin-sort-class-members": "1.4.0",
+    "eslint-plugin-typescript": "0.12.0",
     "merge": "1.2.1",
     "pascal-case": "^2.0.1",
     "pkg-dir": "4.1.0",
-    "pluralize": "^7.0.0"
+    "pluralize": "^7.0.0",
+    "typescript-eslint-parser": "20.0.0"
```

Comment:
Do you know what versions of TypeScript this supports?

Reply:
Yeah it should be on `~3.2.1`, thanks for catching that.

Reply:
Actually, what I had was right. `typescript-estree` doesn't support past `~3.1.1`.